### PR TITLE
re-enable bundle index build for CLO

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:

* Re-enables the bundle index build for the CLO

### CODE REVIEW

File: ci-operator/jobs/infra-image-mirroring-multi-arch.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# DO NOT EDIT. This file is auto-generated.
```
Comment (smg247) [inline ci-operator/jobs/infra-image-mirroring-multi-arch.yaml:1]: Was this intentional?

Comment (jcantrill) [reply ci-operator/jobs/infra-image-mirroring-multi-arch.yaml:1]: No.  I have no idea why this file changed

Comment (jcantrill) [reply ci-operator/jobs/infra-image-mirroring-multi-arch.yaml]: rebased. repushed

**Pull request merged**